app/models/git/track.rb

```diff
@@ -64,13 +42,6 @@ def commit
     private
     attr_reader :repo, :git_sha
 
-    FILEPATHS = {
-      about: "docs/ABOUT.md",
-      snippet: "docs/SNIPPET.txt",
-      debug: "exercises/shared/.docs/debug.md",
-      help: "exercises/shared/.docs/help.md",
-      tests: "exercises/shared/.docs/tests.md",
-      config: "config.json"
-    }.freeze
+    delegate :read_text_blob, :read_json_blob, to: :repo
```

Comment:
Can this be moved into the `Git::HasGitFilepaths` too I wonder?

Reply:
Probably not, for reasons that will become apparent when I will be adding this to the other two Git classes :)